fdbrpc/Locality.cpp

```diff
@@ -171,6 +194,8 @@ ProcessClass::Fitness ProcessClass::machineClassFitness( ClusterRole role ) cons
 			return ProcessClass::OkayFit;
 		case ProcessClass::ProxyClass:
 			return ProcessClass::OkayFit;
+		case ProcessClass::GrvProxyClass:
```

Comment:
If you look at L108 in this file, it is the preference list  for choosing process for a master role. 
Since grvProxy is used to communicate with master and should be not very busy, it may be not so bad to put grvProxy with master on the same process. 

Before your change, we should not because commitProxy is very cpu intensive.